File: 8KYU/Thinkful - NUmber Drills Blue and red marbles/solution.cs
public static class BlueAndRedMarbles {
    public static double GuessBlue(uint blueStart, uint redStart, uint bluePulled, uint redPulled)
    {
        var total = blueStart + redStart;
        var blueRemaining = blueStart - bluePulled;
        return blueRemaining / (double)(total - (bluePulled + redPulled));
    }
}

File: 8KYU/Thinkful - NUmber Drills Blue and red marbles/solution.c
float guess_blue(int blueStart, int redStart, int bluePulled, int redPulled)
{
    int total = blueStart + redStart;
    int blueRemaining = blueStart - bluePulled;
    return blueRemaining / (float)(total - (bluePulled + redPulled));
}